src/CBT.NuGet.Package/version.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://raw.githubusercontent.com/AArnott/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
   "version": "1.0-dev",
-  "buildNumberOffset": 7,
+  "buildNumberOffset": 8,
```

Comment:
You shouldn't need to modify this file.  I only had to do so because I already published NuGet packages and when adopting this versioning system I needed the versions to start higher than what I had already pushed.  Let me know if the versioning isn't working as expected...